monty_hall_paradox/lib/monty_hall_paradox/state.ex

```diff
@@ -2,18 +2,19 @@ defmodule Mhp.State do
   use GenServer
 
```

Comment:
Looking at this again, I wonder if you have two statuses (statii?), one for each individual game, and a second for the overall simulation?